lib/fluent/plugin_helper/http_server/server.rb

```diff
@@ -38,7 +39,7 @@ def initialize(addr:, port:, logger:, default_app: nil, tls_context: nil)
           scheme = tls_context ? 'https' : 'http'
           @uri = URI("#{scheme}://#{@addr}:#{@port}").to_s
           @router = Router.new(default_app)
-          @reactor = Async::Reactor.new(nil, logger: @logger)
+          @reactor = Async::Reactor.new(nil, logger: Fluent::Log::ConsoleAdapter.wrap(@logger))
```

Comment:
Should we support also older Async?

Reply:
We already cut off 1.22 or former:
https://github.com/fluent/fluentd/blob/1e59424f2d1d7f12e20eb5c7223816c1ad9d3b80/fluentd.gemspec#L52